src/main/java/seedu/bankwithus/Account.java

```diff
@@ -0,0 +1,28 @@
+package seedu.bankwithus;
+
+public class Account {
+    protected String name;
+    protected float balance;
```

Comment:
Please make the attributes private and use getters and setters unless there is really strong justification to make it protected. Plus, protected accessibility makes creating packages a pain in the ass